src/modules/temperature.cpp

```diff
@@ -43,10 +44,13 @@ namespace modules {
 
   bool temperature_module::update() {
     m_temp = std::atoi(file_util::contents(m_path).c_str()) / 1000.0f + 0.5f;
+    int m_temp_f = floor(((9.0/5.0) * (m_temp + 32)) + 0.5);
```

Comment:
Seems like the formula is a little off, it should be
`1.8 * temp + 32`

Reply:
Not sure how that's any different.  We do need the floor() and the + 0.5 to round to a whole number.

Reply:
Sorry, I should have been clearer.
You have the `m_temp + 32` in brackets so you multiply that whole thing by 1.8 instead of just `m_temp`.
The rounding part is fine

Reply:
Doh.  Fixing.